GithubIssueTagger/IssueUtilities.cs

```diff
@@ -40,6 +40,23 @@ public static async Task<IEnumerable<Issue>> GetAllIssues(GitHubClient client, s
             return issuesForMilestone;
         }
 
+        public static async Task<IEnumerable<Issue>> GetOpenPriority1Issues(GitHubClient client, string org, string repo)
+        {
+            var nugetRepos = new RepositoryCollection();
+            nugetRepos.Add(org, repo);
+
+            var queryLabels = new string[] { "priority:1" };
+
+            var request = new SearchIssuesRequest()
+            {
+                Repos = nugetRepos,
+                State = ItemState.Open,
+                Labels = queryLabels
+            };
+            var issuesForMilestone = await client.Search.SearchIssues(request);
```

Comment:
SearchIssues is a workaround for the other api we were using.
Had weird JSON deserialization issues, and found this bug: https://github.com/octokit/octokit.net/issues/1762